images/ose-sriov-rdma-cni.yml

```
@@ -0,0 +1,37 @@
+arches:
+- x86_64
+- aarch64
+- ppc64le
+content:
+  source:
+    dockerfile: Dockerfile.ocp
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/rdma-cni.git
+      web: https://github.com/openshift/rdma-cni
+    ci_alignment:
+      streams_prs:
+        ci_build_root:
+          stream: rhel-9-golang-ci-build-root
+dependents:
+- sriov-network-operator
+distgit:
+  branch: rhaos-{MAJOR}.{MINOR}-rhel-9
+  component: ose-sriov-rdma-cni-container
+enabled_repos:
+- rhel-9-appstream-rpms
+- rhel-9-baseos-rpms
+for_payload: false
+from:
+  builder:
+  - stream: rhel-8-golang
+  - stream: rhel-9-golang
+  member: openshift-enterprise-base-rhel9
+name: openshift/ose-sriov-rdma-cni-rhel9
+name_in_bundle: ose-sriov-rdma-cni
```

```suggestion
name_in_bundle: rdma-cni
```
See https://github.com/openshift-eng/ocp-build-data/pull/5162